<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesWidget</class>
 <widget class="QWidget" name="FilesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>587</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView {
	border: 0px;
}

#files {
	color: #395265;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: #ffffff;
	color: rgb(27, 105, 165);
	border-bottom: 1px solid rgb(248, 155, 53);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string notr="true">384 movies</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MyTableView" name="files">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">/*alternate-background-color: rgb(243, 246, 250);*/
selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #4185b6, stop:1 #1b6aa5);
alternate-background-color: #f9f9f9;
selection-color: #ffffff;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="useSearchOverlay" stdset="0">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>22</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>22</number>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">/*
QWidget {
	border: 0px;
	border-right: 1px solid rgba(0, 0, 0, 100);
	background-color: #b0b0b0;
}
*/

border-top: 1px solid rgb(248, 155, 53);
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="MyLabel" name="sortByNew">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f9f9f9;
background-color: #ffffff;
color: rgb(27, 105, 165);
border: none;
border-right: 1px solid #f0f0f0;
font-weight: bold;
font-size: 11px;
padding-top: 4px;
padding-bottom: 4px;
margin-top: 1px;
margin-bottom: 1px;
</string>
        </property>
        <property name="text">
         <string>New</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="sortByName">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f9f9f9;
background-color: #ffffff;
color: rgb(27, 105, 165);
border: none;
border-right: 1px solid #f0f0f0;
font-weight: normal;
font-size: 11px;
padding-top: 4px;
padding-bottom: 4px;
margin-top: 1px;
margin-bottom: 1px;
</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="sortByLastAdded">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f9f9f9;
background-color: #ffffff;
color: rgb(27, 105, 165);
border: none;
border-right: 1px solid #f0f0f0;
font-weight: normal;
font-size: 11px;
padding-top: 4px;
padding-bottom: 4px;
margin-top: 1px;
margin-bottom: 1px;
</string>
        </property>
        <property name="text">
         <string>Date Added</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="sortBySeen">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f9f9f9;
background-color: #ffffff;
color: rgb(27, 105, 165);
border: none;
border-right: 1px solid #f0f0f0;
font-weight: normal;
font-size: 11px;
padding-top: 4px;
padding-bottom: 4px;
margin-top: 1px;
margin-bottom: 1px;
</string>
        </property>
        <property name="text">
         <string>Seen</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="sortByYear">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f9f9f9;
background-color: #ffffff;
color: rgb(27, 105, 165);
border: none;
border-right: 1px solid #f0f0f0;
font-weight: normal;
font-size: 11px;
padding-top: 4px;
padding-bottom: 4px;
margin-top: 1px;
margin-bottom: 1px;
</string>
        </property>
        <property name="text">
         <string>Year</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>smallWidgets/MyLabel.h</header>
  </customwidget>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>smallWidgets/MyTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <signal>movieSelected(Movie*)</signal>
  <signal>noMovieSelected()</signal>
 </slots>
</ui>
